codec-http2/src/main/java/io/netty/handler/codec/http2/InboundHttp2ToHttpAdapter.java

```diff
@@ -182,7 +180,8 @@ protected FullHttpMessage newMessage(Http2Stream stream, Http2Headers headers, b
      * @throws Http2Exception If the stream id is not in the correct state to process the headers request
      */
     protected FullHttpMessage processHeadersBegin(ChannelHandlerContext ctx, Http2Stream stream, Http2Headers headers,
-                boolean endOfStream, boolean allowAppend, boolean appendToTrailer) throws Http2Exception {
+                                                  boolean endOfStream, boolean allowAppend, boolean appendToTrailer) 
```

Comment:
Checkstyle complains about a trailing whitespace on this line.

Reply:
Fixed